### PR TITLE
Don't mask plain key occurrences as it can mask useful log information

### PR DESCRIPTION
**Pull request by liamkeaton**

### CODE REVIEW

**Comment by liamkeaton** [comment]
@alexnaish @remybach you will know more about this, was there any reason why this was done originally?

**Comment by alexnaish** [comment]
Originally it was better to be safer than sorry iirc, it was trying to match the string based on `key=value` but I didn't want to have checks for different kinds of string patterns

File: src/main.js [hunk 1]
```diff
@@ -64,7 +64,7 @@ export default class {
 
 	maskString (rawString) {
 		return rawString.replace(this.sensitiveFields, (match, p1) => {
-			return p1 ? `${p1}="${MASK_SEQUENCE}"` : MASK_SEQUENCE;
+			return p1 ? `${p1}="${MASK_SEQUENCE}"` : match;
```
Comment (CourtneyLO) [inline src/main.js]: I know you did not write this but what does p1 mean?

Comment (alexnaish) [reply src/main.js]: The nth string found by a parenthesized capture group - so if we had a regex like `(.*)=(.*)`, p1 would be the first, p2 would be the second:

`email=test` -> `p1` is `email`

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace#Specifying_a_function_as_a_parameter

Comment (CLTPayne) [reply src/main.js]: So the `pn` represents parenthesis `n` of the regex? (Trying to work out why 'p').

**Comment by liamkeaton** [comment]
Closing in favour of the rewrite https://github.com/Financial-Times/n-mask-logger/pull/29

**Pull request closed without merging**